### ISSUE TITLE
Question about "Pathways" columns in output

### ISSUE THREAD

**Issue by evatosco** [casanova-lab/NHC#3]
Hello!

sorry to show up again, I only seem to bring bad news!

I have been using NHC again after quite a while. After processing a new dataset of cases and controls, I have noticed that in the output results, the pathways columns (Pathways | Pathway_List | Top_Pathway) show no values (zeros or missing ".").

I've checked previous outputs from earlier runs, and sometimes they showed REACTOME or KEGG top pathways. But I wonder if you would consider it "normal" that now, between 232 clusters, not a single one has a top pathway value or even a total number of pathways (even though I have GO pathways annotated for almost all of the output clusters). Can you tell me if this is normal and can happen, or if I'm doing something wrong?

My commands for running NHC are as simple as possible:
```
python3 $NHC/NHC_case_only.py -case $CASES
python3 $NHC/NHC_case_control.py -case $CASES -ctl $CTRLS -pc $PC
```

Also, I have run the test data of the GitHub repo. The output in my machine shows the same missing values for said columns for all clusters, so I guess it can be possible, but I wanted to ask you anyway, since it also happened but with a much larger number of clusters.

Here's a peak of the mentioned columns in my output file:
#Pathways | Pathway_List | Top_Pathway | GO_BP_List | GO_MF_List
0 | . | . | GO:0000082:G1/S transition of mitotic cell cycle(1.807e-24),GO:0006270:DNA replication initiation(2.376e-16),GO:0006260:DNA replication(3.547e-15),GO:0006267:pre-replicative complex assembly involved in nuclear cell cycle DNA replication(2.671e-09),GO:0000727:double-strand break repair via break-induced replication(3.028e-06)
0 | . | . | .
0 | . | . | .
0 | . | . | .
0 | . | . | GO:0042267:natural killer cell mediated cytotoxicity(1.181e-10)
0 | . | . | .
0 | . | . | GO:0017183:peptidyl-diphthamide biosynthetic process from peptidyl-histidine(6.196e-06)
0 | . | . | .
0 | . | . | GO:0006414:translational elongation(2.064e-12)
0 | . | . | GO:0006777:Mo-molybdopterin cofactor biosynthetic process(2.202e-09),GO:0032324:molybdopterin cofactor biosynthetic process(2.202e-09)

If you think I should check something else, please tell me.
Thank you so much in advance!

**Comment by zhangpeng1202** [comment]
Hi, Eva,
Thank you very much for your feedback! I am sorry for the issue. I recently updated the background network and statistical test in the code. Let me look into the problem, and get back to you asap (by this weekend).

Best,
Peng

**Comment by zhangpeng1202** [comment]
Hi, Eva,

Please try again. Let me know if it works now.
One column of the pathway data was shifted, which caused the problem.
I also added GSEA Hallmark genesets into pathway data, as an additional enrichment resource.
Thanks again!

Best,
Peng

**Comment by evatosco** [comment]
Hi again!

Thank you so much for your fast support! 
I genuinely thought I was doing something wrong, but no problem. I am glad it could be solved. These issues where the columns jump over are so annoying, it is so simple yet affects so much. I understand.

I tried the test data again, and the output shows REACTOME and KEGG and other values of annotation again. So everything is okay now.
I am currently rerunning my data. It takes a while, but after running the test, I know it will be correct once it finishes.

Closing this issue since I think it is already solved!
Thanks again!
